Evaluation/Workflow.m

```diff
@@ -6,62 +6,68 @@
+% - ReportOutput_path: final report will be generated here
+qualificationPlan = fullfile(basisDir,'input',qualificationPlanName);
+REInput_path = fullfile(basisDir,'re_input');
+REOutput_path = fullfile(basisDir,'re_output');
+ReportOutput_path=fullfile(basisDir,'report');
 
 % --------------------------------------------------------------
-%OPTIONAL: replace qualificationRunnerPath and qualificationPlan with your paths and call the qualification runner first
-qualificationRunnerPath = fullfile(cd,'..','..','..','_software','QualificationRunner');
-qualificationPlan = fullfile(cd,'Input','evaluation_plan.json');
-
-startQualificationRunner(qualificationRunnerPath, qualificationPlan, REInput_path);
+% STEP #1: start qualification runner to generate inputs for the reporting engine
+startQualificationRunner(qualificationRunnerFolder, qualificationPlan, REInput_path);
 
 % --------------------------------------------------------------
+% STEP #2: start reporting engine
 % Get the Configuration Plan Settings
-[WSettings, ConfigurationPlan, TaskList, ObservedDataSets] = initializeQualificationWorkflow(jsonFile, REInput_path, REOutput_path);
+reportConfigurationPlan = 'report-configuration-plan.json';
+[WSettings, ConfigurationPlan, TaskList, ObservedDataSets] = initializeQualificationWorkflow(reportConfigurationPlan, REInput_path, REOutput_path);
 
 %OPTIONAL: set watermark. If set, it will appear in all generated plots
 %WSettings.Watermark = 'Preliminary';
 
-% --------------------------------------------------------------
 % run the Worklfow tasklist of ConfigurationPlan
 runQualificationWorkflow(WSettings, ConfigurationPlan, TaskList, ObservedDataSets);
 
 QualificationWorkflowTime = toc/60;
 fprintf('\n Qualification Workflow Duration: %0.1f minutes \n', QualificationWorkflowTime);
 
 % --------------------------------------------------------------
-%OPTIONAL: call MarkdownJoiner to combine Reporting Engine output into the final report
-% (this can be also done later from Windows command line)
-%
-% replace ReportOutput_path and MarkdownJoiner_path with your paths
-%
-% - ReportOutput_path: final report will be generated here
-%
-% - MarkdownJoiner_path: location of markdown-joiner.exe
-
-ReportOutput_path=fullfile(cd,'Output','md_output');
-MarkdownJoiner_path=fullfile(cd,'..','..','..','_software','markdown-joiner','markdown-joiner.exe');
+% STEP #3: call MarkdownJoiner to combine Reporting Engine output into the final report
+MarkdownJoiner_path=fullfile(markdownJoinerFolder,'markdown-joiner.exe');
 
 % alternative #1: ReportOutput_path must be empty. If not, report generation will fail
 status = system([MarkdownJoiner_path ' -i ' REOutput_path ' -o ' ReportOutput_path]);
 
 % alternative #2: (CAUTION) ReportOutput_path will be cleared first
 %status = system([MarkdownJoiner_path ' -i ' REOutput_path ' -o ' ReportOutput_path ' -f']);
 
-if status~=0
-    error('MarkdownJoiner failed');
-end
+if status~=0 error('MarkdownJoiner failed'); end
```

Comment:
Must write the if-statement in one line in order to let **compiled version** of RE work